.github/workflows/build.yaml

```diff
@@ -54,3 +54,61 @@ jobs:
         run: "make generate \nif ! git diff --exit-code --quiet ; then \n  echo\n  echo\n  git diff --stat \"HEAD\"\n  echo\n  echo\n  echo 'ERROR: Lint tools caused changes to the working dir. '\n  exit 1\nfi\n"
       - name: make test
         run: make test
+  e2e:
```

Comment:
nit: thoughts on re-naming this file `tests.yaml` and the [top level name](https://github.com/GoogleCloudPlatform/cloud-sql-proxy-operator/blob/main/.github/workflows/build.yaml#L15) to `tests`?

Reply:
Done.

Reply:
awesome, FYI you will need to change the required check `build and unit test` to not be required prior to merging and then after merging you will want to add the `unit tests` as a required check in its place.